data/coalition/coalition culture conversations.txt

```diff
@@ -243,12 +243,31 @@ mission "Coalition Departing Trouble - A"
 	source
 		government "Coalition"
 		not attributes "coalition station"
-	destination "Ring of Friendship"
 	on enter
+		fail
 		conversation
 			`Your ship narrowly avoids a few small Coalition vessels as you exit <origin>'s atmosphere. Not a second later, while you're still stunned by the near-crash, once again the <ship> is almost hit by incoming ships, this time Heliarch warships, in what looks like an intense chase after the ships from before.`
-			`	After they've all passed, you check your comms and radar, finding some other civilian ships nearby seem to have experienced a similar shock of their own. Some of the ships linger over <origin> for a while, as if unsure of what to do. Others bolt away with engines at full power, leaving the system in a blink, and some eventually decide to land again, either too curious about the situation, or perhaps wanting to avoid any trouble.`
+			`	After they've all passed, you check your comms and radar, finding some other civilian ships nearby that seem to have experienced a similar shock of their own. Some of the ships linger over <origin> for a while, as if unsure of what to do. Others bolt away with engines at full power, leaving the system in a blink, and some eventually decide to land again, either too curious about the situation, or perhaps wanting to avoid any trouble.`
+			branch heliarch
+				has "joined the heliarchs"
+			branch lunarium
+				has "joined the lunarium"
 			`	Some more minutes later, you see a select few of the Heliarch ships that passed close to you departing the planet. They pay you no mind, and much to your relief steer a safe distance from your ship this time.`
+				decline
+			label heliarch
+			`	Some more minutes later, you see a select few of the Heliarch ships that passed close to you departing the planet, steering a safe distance from your ship this time. The captain of one of the ships, a Heliarch Interdictor, hails you, apologizing for the earlier distress, briefly mentioning they've gotten the situation back in control. They wish you a good day, and prepare to leave the system.`
+				decline
+			label lunarium
+			`	Some more minutes later, you see a select few of the Heliarch ships that passed close to you departing the planet. They pay you no mind, and much to your relief steer a safe distance from your ship this time. You cannot help but wonder whether the ships they were chasing held any Lunarium members, or if they were simply more unfortunate civilians.`
```

Comment:
```suggestion
			`	Some more minutes later, you see a select few of the Heliarch ships that passed close to you departing the planet. They pay you no mind, and much to your relief steer to a safe distance from your ship this time. You cannot help but wonder whether the ships they were chasing held any Lunarium members, or if they were simply more unfortunate civilians.`
```

Reply:
```suggestion
			`	Some more minutes later, you see a select few of the Heliarch ships that passed close to you departing the planet. They pay you no mind, and much to your relief steer a safe distance away from your ship this time. You cannot help but wonder whether the ships they were chasing held any Lunarium members, or if they were simply more unfortunate civilians.`
```
Alternative idea.

Reply:
(Honestly, I'm not 100% sure this is needed at all.)

Reply:
I don't really mind either way, so just tell me when you've decided on one of the changes or no changes to this.

Reply:
I prefer the second one.

Reply:
Let's go with the second one, then.